command/node_status.go

```diff
@@ -170,6 +180,29 @@ func (c *NodeStatusCommand) Run(args []string) int {
 			return 1
 		}
 
+		// Return nothing if no nodes found
+		if len(nodes) == 0 {
+			return 0
+		}
+
+		var size int
+		if c.quiet {
+			size = len(nodes)
+		} else {
+			size = len(nodes) + 1
+		}
+
+		// Format the nodes list
+		out := make([]string, size)
+
+		if c.quiet {
+			for i, node := range nodes {
+				out[i] = limit(node.ID, c.length)
```

Comment:
Hm, now that I'm seeing this, do we necessarily want to limit `-quiet` to the short length? You can't use those short IDs in API inputs in a lot of cases, so it might be nice to always show the full UUID.

Reply:
I was perplexed about this too! Thanks for clarifying it. Changed it to show full UUID.